src/System.IO.FileSystem/tests/Performance/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard-Windows_NT;
```

Comment:
Why Windows specific -- because our perf tests are currenlty Windows specific? These guys are all just `netstandard`:

C:\git\corefx\src\System.IO.Pipes\tests\Performance\Configurations.props
C:\git\corefx\src\System.Runtime.Serialization.Json\tests\Performance\Configurations.props
C:\git\corefx\src\System.Runtime.Serialization.Xml\tests\Performance\Configurations.props